libnopegl/src/node_text.c

```diff
@@ -325,6 +325,9 @@ static int update_character_geometries(struct ngl_node *node)
             ngli_pipeline_compat_update_attribute(desc->pipeline_compat, 3, s->transforms);
 
             ngli_pipeline_compat_update_attribute(desc->pipeline_compat, 4, s->atlas_coords);
+
+            if (s->text_ctx->cls->mutable_texture)
+                ngli_pipeline_compat_update_texture(desc->pipeline_compat, 0, s->text_ctx->texture);
```

Comment:
not related but can you add a separate commit to move the ngli_buffer_upload() block below just after the ngli_buffer_init() block above ?

Reply:
That's ok as is for now.